contracts/CourtRoom.sol

```diff
@@ -220,12 +277,12 @@ contract SwearGame is Owned {
 			ens = ENSAbstract(ensAddress);
 			bytes32 contentHash1 = ensResolve(clientENSNameHash);
 			bytes32 contentHash2 = ensResolve(reflectorENSNameHash);
-			bool res = (contentHash1 == contentHash2);
+			bool res = (contentHash1 != contentHash2);
 			if (res){
-				Decision("not guilty");
+				Decision("guilty");
```

Comment:
i would have return values the other way in line with common bool cast (0=false) 
answering the question "guilty?"
(also in line with potentially multiple non-zero values for various levels of guilt/punishment in future, much like shell exit codes)